src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -74,7 +74,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $output = new SymfonyStyle($input, $output);
+        $output = new SymfonyStyle($input, $cliOutput = $output);
```

Comment:
I don't really care, but this may not be super clear to people reading the code - maybe we can set it in its own line